mw-migrations/src/migrations/dayReportsMigration.ts

```diff
@@ -35,27 +34,49 @@ const migrateDayReports = async () => {
       log(`started ${dayReport.uuid} migration`);
       
       const dayReportRef = doc(db, "dayReports", dayReport.uuid);
-      const jobsDonePromise = Promise.all(dayReport.jobDoneUuids.map(JobDoneService.getJobDoneDTO));
-      const plansPromise = Promise.all(dayReport.planForNextPeriodUuids.map((PlanForNextPeriodService.getPlanForNextPeriodDTO)));
-      const problemsPromise = Promise.all(dayReport.problemForCurrentPeriodUuids.map(CurrentProblemService.getProblemDTO));
-      const commentsPromise = Promise.all(dayReport.commentUuids.map(CommentService.getCommentDTO));
 
-      const [jobsDone, plans, problems, comments] = await Promise.all([jobsDonePromise, plansPromise, problemsPromise, commentsPromise]);
+      const jobsDone: JobDoneDTOMigration[] = dayReport.jobsDoneStringified.map((jobDone) => JSON.parse(jobDone));
+      const plans: PlanForNextPeriodDTOMigration[] = dayReport.plansStringified.map((plan) => JSON.parse(plan));
+      const problems: CurrentProblemDTOMigration[] = dayReport.problemsStringified.map((problem) => JSON.parse(problem));
 
-      batch.set(dayReportRef,
+      const jobsDoneUpdated = jobsDone.map((jobDone) => {
+        return {
+          ...jobDone,
+          tags: jobDone.tags.length === 0
```

Comment:
OCreate a function and should simplify, the same everywhere

Reply:
I don't think that it is needed for migration

Reply:
Yeah, rigth